source/_integrations/amcrest.markdown

```diff
@@ -205,6 +206,129 @@ Service data attribute | Optional | Description
 -|-|-
 `entity_id` | no | Name(s) of entities, e.g., `camera.living_room_camera`.
 
+#### Service `ptz_control`
+
+If your Amcrest or Dahua camera supports PT/PTZ, you will be able to pan, tilt or zoom your camera. Zoom is only supported on PTZ
+cameras with optical zoom. It does not support zooming of Varifocal lenses.
+
+| Service data attribute | Description |
+| -----------------------| ----------- |
+| `entity_id` | String or list of strings that point at `entity_id`s of cameras. Use `entity_id: all` to target all. |
```

Comment:
This should be consistent with the `entity_id` description of all the other services. Either change them all or make this one the same as the others. It would be nice if you changed them all to:

The entity ID of the camera to control. May be a list of multiple entity IDs. To target all cameras, set entity ID to `all`.

Reply:
changed them all to your suggestion.